jaseci_kit/jaseci_kit/modules/text_segmenter/text_segmenter.yaml

```diff
@@ -22,7 +22,7 @@ data:
     cd /jaseci/jaseci_core/
     git checkout text_segmentation_prod
     source install.sh
-    cd /jaseci/jaseci_kit/jaseci_kit/modules/text_segmenter
+    cd /jaseci/jskit/text_segmenter
```

Comment:
Hi @Gorgeous-Patrick I dont think this path would work with the latest main branch. Take a look.

Reply:
Well, yes it does not work for the main branch. And the main repo does not have that `text_segmentation_prod` branch actually. 
The other way around is to use the programs and paths in the main branch in the main repo. I just thought that maybe the programs are different (the branch is named a production branch so it looks like a more mature version). @marsninja

Reply:
Not sure why this manifest was pointing to Mahendra's fork instead of the main repo. I have corrected it and it should work now.